src/pages/_app.tsx

```diff
@@ -64,7 +64,12 @@ const MyApp: AppType<{ session: Session | null }> = ({
         <meta property="og:image" content="https://splitpro.app/og_banner.png" />
       </Head>
       <SessionProvider session={session}>
-        <ThemeProvider attribute="class" defaultTheme="dark">
+        <ThemeProvider
```

Comment:
it is best to do this in separate PR and on option to swithch in account settings

Reply:
Of course! I can revert it and then make another or for that!
A setting in settings page for either "light, dark or system prefer" would be the best option!
I will take a look at it

Reply:
legend thanks!

Reply:
Here is a new PR: https://github.com/oss-apps/split-pro/pull/133